build/build-config.yml

```diff
@@ -26,3 +26,8 @@ config:
         image-name: "estimate-service"
 
 # frontend
+  - name: builds/digit-works/frontend/micro-ui/works
+    build:
+      - work-dir: frontend/micro-ui/
```

Comment:
Why does work-dir does not contain /works but name does contain micro-ui/works? 
Is it by mistake?

Reply:
in previous repo Digit-dev also we have given. the build image 
name over there